apb/templates/broker-user-auth.yaml

```diff
@@ -0,0 +1,17 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: asb-user-access
```

Comment:
I think this name, since it's cluster scope, should be related to the `{{ broker_name }}`

Reply:
or...maybe not, but at least not `asb-user-access` :sunglasses: